lib/remap-lang/grammar.pest

```diff
@@ -109,7 +109,7 @@ lookup         = ${
   // Other paths (internal to Vector) do not necessarily have this distinction, despite mapping the same way.
   | (lookup_segment ~ (LOOKUP_PERIOD ~ lookup_segment)*)
 }
-lookup_segment = ${ ((lookup_field | lookup_field_quoted | lookup_array) ~ lookup_array*) | lookup_coalesce }
```

Comment:
Ahh omg I hadn't considered this! Does it just slip right in?

Reply:
Yeah. just works. Thinking about it, it could probably be simplified even further..